### DISCUSSION TITLE
Streamlink not these channels

### DISCUSSION THREAD

**Discussion by Nowadaysix** [iptv-org/community#159] [Q&A]
It always gives an error when trying to use Streamlink. Example using the SBT RJ Channel - Brazil

https://streamlink-iptv.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/c/SBTRJ/live

SBT RJ Brazil
https://www.youtube.com/c/SBTRJ/live
SCC SBT Brazil 
https://www.youtube.com/c/SCCSBT/live
SBT RS Brazil 
https://www.youtube.com/c/sbtrs/live
VTV SBT Brazil 
https://www.youtube.com/c/JornalismoVTVSBT/live

All streams are public and still the strealink doesn't work. Does anyone have a solution to make it work?

**Comment by Central302** [comment]
[SBT RJ]Link is Geo-Blocked.
[SCCSBT]I Can't Find The Stream (Link Was Redirected to Channel Description)
[SBT RS]Link Is Geo-Blocked.
[VTV]Link is Geo-Blocked.
 
[Site Issue?]dct-inftA Version of Proxy also Can't Get The Link. but Other Youtube Stream Works.

**Reply by BellezaEmporium** [reply]
If link is geoblocked it'll simply won't work.
